config/routes.rb

```diff
@@ -3,6 +3,7 @@
 root to: "urls#index"
 
 resources :urls
+get ':slug' => 'urls#show'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.